.teamcity/settings.kts

```diff
@@ -176,7 +175,8 @@ object RunAllUnitTests : BuildType({
 				export NODE_ENV="test"
 
 				# Update node
-				. "${'$'}NVM_DIR/nvm.sh"
+				. "${'$'}NVM_DIR/nvm.sh" --no-use
```

Comment:
I'm not 100% sure why, but the previous code was not picking the new node version. I tested this config using TeamCity custom builds + a patch, and it seems to work.